public/locales/en/translation.json

```diff
@@ -336,13 +336,15 @@
           "singular": "some geometries"
         }
       },
-      "label": "Enter coordinates:",
+      "label": "Or enter coordinates",
+      "legend": "Coordinates",
       "title": "Coordinates (decimal WGS84)",
       "latitude": "Latitude",
       "longitude": "Longitude",
-      "helper-create": "Draw $t(jsonSchema.geometryField.type.{{type}}.singular) with available button on map.",
-      "helper-edit": "Edit $t(jsonSchema.geometryField.type.{{type}}.singular) with available button on map.",
-      "save": "Once edited, click on submit button."
+      "information-create": "Or draw $t(jsonSchema.geometryField.type.{{type}}.singular) on map.",
```

Comment:
Can you simplify the interpolate var ?

Reply:
What do you think? 
According to `type`, it should pick the correct wording between lines 317 and 337